compiler/ballerina-lang/src/test/java/io/ballerina/projects/DependenciesTomlTests.java

```diff
@@ -282,6 +282,44 @@ public void testDependenciesTomlWithoutRootPackage() throws IOException {
                         "Detected corrupted Dependencies.toml file. This will be updated to latest dependencies.");
     }
 
+    /**
+     * Invalid Dependencies.toml file with additional attribute in modules array.
+     */
+    @Test
```

Comment:
Shall we use the `description` attribute instead, as follows? 
```suggestion
    @Test(description = "Invalid Dependencies.toml file with additional attribute in modules array.")
```

Reply:
Added via 7fb09ce97a2882bd998fe12cdba5d5339db52b31